### ISSUE TITLE
Setup Script

### ISSUE THREAD

**Issue by maxwnewcomer** [maxwnewcomer/NWTOPT#22]
Would like to create a setup script that automatically does the following:

conda env create -n <env_name> -f conda_requirements.yml
conda activate <env_name>
conda pack -n <env_name> -o nwtenv.tgz
mkdir mongodb/db
mkdir logs
mkdir logs/condor_logs; mkdir logs/errors; mkdir logs/outputs

named: setup.sh

**Comment by mnfienen** [comment]
@maxwnewcomer --- so this will happen once on the submitter and then the env made by `conda pack` will get deployed? If I understand that, I say thumbs up!

**Comment by maxwnewcomer** [comment]
@mnfienen exactly! basically just to make fresh installs easier. i was working on new run and had to go through documentation to get my clone working and realized everything would be much easier if it was all just a script. leaving this issue for chris as it will be a good intro to using github, pull requests, bash scripts, and to familiarize him with nwtopt a little bit

**Comment by mnfienen** [comment]
yes and yes!
